src/app/lib/presentation/router_examples/setting_page.dart

```diff
@@ -1,4 +1,3 @@
-import 'package:app/app_router.dart';
 import 'package:flutter/material.dart';
```

Comment:
@jeanplevesque Est-ce que tu penses que je devrais juste completement delete cette page?

Reply:
I don't understand what router_examples is?

Reply:
Basically, router_examples was just a section I added to put pages in it that I added to show richer examples of navigation (there are only two pages in the app apart from theses pages.)

I was thinking maybe it would be best to remove the "base page" with the menu and keep the others while there is still no real navigation aside from the menu?

Reply:
Let's remove it. The app will have more pages eventually.